accounts/abi/event_test.go

```diff
@@ -54,3 +85,80 @@ func TestEventId(t *testing.T) {
 		}
 	}
 }
+
+func TestEventTupleUnpack(t *testing.T) {
+	type EventTransfer struct {
+		Value *big.Int
+	}
+
+	type EventPledge struct {
+		Who      common.Address
+		Wad      *big.Int
+		Currency [3]byte
+	}
+
+	var testCases = []struct {
+		data     string
+		dest     interface{}
+		expected interface{}
+		jsonLog  []byte
+		name     string
+	}{{
```

Comment:
Forgot to ask in my previous review: are you going to test for errors as well? Like, the array boundary check that you made some changes for?